albatross/core/cereal_utils.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2018 Swift Navigation Inc.
+ * Contact: Swift Navigation <dev@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#ifndef ALBATROSS_CEREAL_UTILS_H
+#define ALBATROSS_CEREAL_UTILS_H
+
```

Comment:
NIT: Maybe this file is `cereal_eigen.h`? It serves a very clear purpose currently and it might be nice to codify that in the name.

Reply:
I moved it to `/cereal/eigen.h`.  That sound good?